CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Change Log
 
+Version 0.18.0 *(2021-09-13)*
+---------------------------------
+
+- **Behavior Change:** When using version 7.1.0 or newer of the Android Gradle Plugin we will now publish all variants
```

Comment:
Can we outline what exactly changed? For instance gradle task names

Reply:
The main task names `publish`, `publishToMavelLocal`, `publishAllPublicationsTo[Name]Repository` stay the same as before. I will take a look if one of the more granular ones changes. In general this more of a change in what is published for an Android module.